test/extended/operators/routable.go

```diff
@@ -33,7 +33,9 @@ var _ = g.Describe("[sig-arch] Managed cluster should [apigroup:apps.openshift.i
 	)
 
 	g.BeforeEach(func() {
-		_, ns, err := exutil.GetRouterPodTemplate(oc)
+		apiResourceExistsFn, err := exutil.DoesApiResourceExistCached(oc.AdminConfig())
+		o.Expect(err).NotTo(o.HaveOccurred())
+		_, ns, err := exutil.GetRouterPodTemplate(oc, apiResourceExistsFn)
```

Comment:
in case we are not okay doing so many changes to other tests than `should serve routes that were created from an ingress`, we can always pass a `nil` in tests like these instead ,for the old behaviour of always checking `DeploymentConfigs`